demo/README.md

```diff
@@ -59,40 +57,60 @@ $ sudo docker rmi <image-id>
+# Service info setup between FIDO IoT HTTP Java Device Sample and FIDO IoT Owner Sample
 
-`$ keytool -importkeystore -destkeystore path/to/dest-keystore.p12 -srckeystore src-keystore.p12 -srcstoretype PKCS12 -alias newkeypair`
+The FIDO IoT HTTP Java Device Sample curretly supports `sdo_sys` module for interpreting received owner service info and `devmod` module to share device service info with Owner.
+
+- `sdo_sys` Owner service info module: This module supports the following 3 message names as listed below to interpret the service info as received from the Owner. The basic functionality of this module is to support payload/script transfers and basic command execution.  A sample format looks like 'sdo_sys:filedesc=filename, sdo_sys:write=filecontent,sdo_sys:exec=command-to-execute'.
+
+    *filedesc* - The name to be given to the file once it is transferred. Upon receiving this, device creates a file with the given name and opens stream to write into it.
+
+    *write* - The payload/content (script, binaries, and others) that is sent to the device. Upon receiving this, device writes the content into the open stream as given by the preceeding 'filedesc' message.
+
+    *exec* - The command that will be executed at the device. Device executes the command as received.
+
+***NOTE*** The comma-separated values must be ordered such that the 'filedesc' and 'write' objects are one after the other pair-wise, followed by the 'exec' commands.
+
+- `devmod` Device service info module: This module supports multiple messages as listed down in the protocol specification, that are sent to the Owner as Device Service info. A sample format looks like 'devmod:active=1'.
+
+The FIDO IoT Owner Sample currently supports the same `sdo_sys` module to send Owner service info to the Device and `devmod` module to store the received Device Service info. Few sample service info values, as present in \<fido-iot-src\>/demo/owner/serviceinfo/sample-values/ are populated into the database table 'OWNER_SERVICEINFO' as byte arrays. For keeping the association between the Device and the service info values to transfer, 'GUID_OWNERSVI' database table is used. When a Device is inserted into the database table 'TO2_DEVICES', it'll not have any association with the service info values, and so by default, no service info is transferred to the Device.
+
+# Enabling Service info transfer
+
+To enable service info transfer to a Device with a given GUID, following are the steps:
+
+1. (Optional) Insert required Service info values into the database table 'OWNER_SERVICEINFO' using the API `POST /api/v1/owner/svivalues/?id=<ServiceInfoId>&isCborEncoded=<boolean>`. More information about the same is provided in section [FIDO IoT Owner REST APIs](owner/README.md/#fido-iot-owner-rest-apis). If the required service info already exists in the table, go on to the next step.
```

Comment:
Missed out this in the earlier PR. Please update the API's supposed values <ServiceInfoId> & <boolean> as per Owner's REST APIs. Same for <guid> in the next point

Reply:
Done